portfolio/src/main/webapp/index.html

```diff
@@ -50,14 +50,23 @@ <h3>Projects and Hackathons</h3>
         <dd>Collaborated with a team of 3 students to design an application for American Airlines to improve customer service for passengers with disabilities.
         We developed our prototype in Figma to demo during the pitch.</dd>
       </dl>
+    </div>
+
+    <div class="comment-area">
       <form action="/data" method="POST">
         <h3>Comments</h3>
-        <textarea name="text-input"></textarea>
+        <textarea name="text-input" placeholder="Type Here"></textarea>
         <br/><br/>
+    
 
-        <input type="submit" />
+        <input type="Submit"/>
```

Comment:
I believe this should actually be lower case "s" `submit`

Reply:
Done.